groups/script.js

```diff
@@ -102,6 +111,38 @@ groupRoles?.addEventListener('click', function (event) {
   }
 });
 
+/**
+ * FOR SEARCHING A GROUP
+ */
```

Comment:
I don't think these types of comments are important.

Reply:
Removed comments